source/user-manual/reference/internal-options.rst

```diff
@@ -986,9 +986,9 @@ Syscheck
 |                                      |               |                                                                                |
 |                                      |               | .. versionadded:: 3.4.0                                                        |
 +                                      +---------------+--------------------------------------------------------------------------------+
-|                                      | Default value | 10                                                                             |
+|                                      | Default value | 5                                                                              |
```

Comment:
In a 3.12.3 fresh installation:

```
root@1312f6d66228:/# cat /etc/ossec-init.conf 
DIRECTORY="/var/ossec"
NAME="Wazuh"
VERSION="v3.12.3"
REVISION="31209"
DATE="Wed May  6 13:53:50 UTC 2020"
TYPE="agent"

root@1312f6d66228:/# cat /var/ossec/etc/internal_options.conf | grep "syscheck.rt_delay"
syscheck.rt_delay=10
```

Are we sure that this value must be changed?

Reply:
Sorry, wrong base branch. This change are to 3.13 release